server/apiHandlers.go

```diff
@@ -17,13 +17,25 @@ package main
 // This is what implements the per-request logic for each API method.
 
 import (
+	"github.com/golang/glog"
+
 	"google.golang.org/genproto/googleapis/rpc/code"
 	"google.golang.org/genproto/googleapis/rpc/status"
 
 	"istio.io/mixer/adapters"
 	"istio.io/mixer/api/v1"
 )
 
+const (
```

Comment:
I created and put them in DispatchKey.go. Would that work? (Will send the update soon).